huxley/www/static/js/huxley/components/AdvisorAssignmentsView.js

```diff
@@ -56,11 +60,33 @@ var AdvisorAssignmentsView = React.createClass({
     }.bind(this));
     CountryStore.getCountries(function(countries) {
       var new_countries = {};
-      for (var i = 0; i <countries.length; i++) {
+      for (var i = 0; i < countries.length; i++) {
         new_countries[countries[i].id] = countries[i];
       }
       this.setState({countries: new_countries})
     }.bind(this));
+    DelegateStore.getDelegates(user.school.id, function(delegates) {
```

Comment:
To avoid explicit bind, we can use arrow functions from ES2015:
```js
DelegateStore.getDelegates(user.school.id, (delegates) => {
  // do stuff...
}); // <- no more .bind(this)
```